main.tf

```diff
@@ -18,7 +18,7 @@
  *       source         = "trussworks/logs/aws"
  *       s3_bucket_name = "my-company-aws-logs"
  *       region         = "us-west-2"
- *       expiration     = 90
+ *       s3_log_bucket_retention     = 90
```

Comment:
This is formatted weird now. All the `=` should be aligned. Unfortunately you have to do this by hand since it's in a comment block.